scalecube-configuration/src/main/java/io/scalecube/configuration/ConfigRegistryConfiguration.java

```diff
@@ -34,29 +34,7 @@ public static ConfigRegistry configRegistry() {
             .addLastSource(
                 "classpath", new ClassPathConfigSource(path -> path.toString().endsWith(".props")));
 
-    // for test purposes without vault access
-    if (System.getenv().get("VAULT_ADDR") != null) {

```

Comment:
this block is important, we receive the couchbase settings from the Vault. So revert this block, except for `addVaultApiKeysPath(builder);`.

```java
    // for test purposes without vault access
    if (System.getenv().get("VAULT_ADDR") != null) {
        builder.addLastSource("vault", VaultConfigSource.builder().build());
    }
```